packages/gui/src/components/Editor/Controls.tsx

```diff
@@ -50,9 +53,23 @@ const Control = ({ field }: ControlProps) => {
   return (
     <Component
```

Comment:
I might create a second, wrapped component here. Something like `ComponentWithDependentProperties` so that we're not overloading the simple use case here.

Reply:
Or `ComponentWithPropertyGroup` or something 🤷